public/pages/CreateRollup/utils/helpers.ts

```diff
@@ -41,6 +42,34 @@ export const parseTimeunit = (timeunit: string): string => {
   return timeunit;
 };
 
+export const delayTimeUnitToMS = (delay: number, timeunit: string): number => {
+  if (timeunit == "SECONDS") {
```

Comment:
nit: can extract SECONDS MINUTES... into one place as constants, so that using them would be less error-prone.